ffi/src/sec_handle.rs

```diff
@@ -1029,15 +1029,15 @@ pub unsafe extern "system" fn ChangeAccountPasswordW(
 
         let domain = c_w_str_to_string(psz_domain_name);
         let username = c_w_str_to_string(psz_account_name);
-        let password = c_w_str_to_string(psz_old_password);
-        let new_password = c_w_str_to_string(psz_new_password);
+        let mut password = Secret::new(c_w_str_to_string(psz_old_password));
+        let mut new_password = Secret::new(c_w_str_to_string(psz_new_password));
 
         ChangeAccountPasswordA(
             security_package_name.as_ptr() as *mut _,
             domain.as_ptr() as *mut _,
             username.as_ptr() as *mut _,
-            password.as_ptr() as *mut _,
-            new_password.as_ptr() as *mut _,
+            password.as_mut().as_ptr() as *mut _,
+            new_password.as_mut().as_ptr() as *mut _,
```

Comment:
I believe `as_mut_ptr()` should be used here. Do you know if there is a reason for not doing so?

Reply:
I added using `as_mut_ptr()` here